src/ClientData/include/ClientData/CaptureData.h

```diff
@@ -165,8 +165,7 @@ class CaptureData {
 
   void FilterBrokenCallstacks() { callstack_data_.UpdateCallstackTypeBasedOnMajorityStart(); }
 
-  void AddUniqueTracepointEventInfo(uint64_t key,
-                                    orbit_grpc_protos::TracepointInfo tracepoint_info) {
+  void AddUniqueTracepointEventInfo(uint64_t key, TracepointInfo tracepoint_info) {
```

Comment:
I think that key should be `tracepoint_id`.

Reply:
Done.